### PR TITLE
Fix multiline replies having only one line sent to IRC

### LINKED ISSUE #1198: Multi-line replies aren't forwarded to IRC correctly
**Describe the bug**
If I send a multi-line message, whether it's text or HTML, it gets forwarded as multiple messages on IRC:
```
matrix:
  "content": {
    "body": "a multiline message\ngets sent as two messages on IRC",
    "msgtype": "m.text"
  },
IRC:
[19:22] <nicolas17[m]> a multiline message
[19:22] <nicolas17[m]> gets sent as two messages on IRC
```
```
matrix:
  "content": {
    "body": "multiline with _formatting_\nthis time",
    "format": "org.matrix.custom.html",
    "formatted_body": "multiline with <em>formatting</em><br />this time",
    "msgtype": "m.text"
  },
IRC:
[19:26] <nicolas17[m]> multiline with _formatting_
[19:26] <nicolas17[m]> this time
```
However, **if the message is a reply**, and has multiple lines, only the first line is sent:
```
matrix:
  "content": {
    "body": "> <@nicolas17:kde.org> let me make a better example\n\nmulti-line\nmessage",
    "format": "org.matrix.custom.html",
    "formatted_body": "<mx-reply><blockquote><a href=\"https://matrix.to/#/!jDLOUpjYLfQYvmOufZ:matrix.org/$160686188544695oCasV:kde.org?via=kde.org&via=matrix.org&via=igalia.com\">In reply to</a> <a href=\"https://matrix.to/#/@nicolas17:kde.org\">@nicolas17:kde.org</a><br>let me make a better example</blockquote></mx-reply>multi-line<br />message",
    "m.relates_to": {
      "m.in_reply_to": {
        "event_id": "$160686188544695oCasV:kde.org"
      }
    },
    "msgtype": "m.text"
  },
IRC:
[19:31] <nicolas17[m]> <nicolas17[m] "let me make a better example"> multi-line
```
The second line of the Matrix message never appears on IRC.

**To Reproduce**
1. Go to a Matrix channel bridged to IRC, and connect to the IRC channel directly too
2. On Matrix, click "Reply" on an existing message
3. Type multiple lines in your message (Ctrl-Enter in Element)
4. See what appears on the IRC client

**Expected behavior**
The other lines of the message should be forwarded to IRC too, like it happens for non-reply messages.

### PR DESCRIPTION
**Pull request by tadzik**
Fixes #1198

### CODE REVIEW

**Comment by tadzik** [comment]
These tests really need some refactoring, as with each of these I'm adding to the problem of copypasted tests :) I'll get a round tuit once all these bugfixes are actually in.

**Review by Half-Shot — APPROVED** [review]
Can't see an issue with this :+1:

File: spec/integ/matrix-to-irc.spec.js [hunk 1]
```diff
@@ -534,6 +534,53 @@ describe("Matrix-to-IRC message bridging", function() {
...
+            origin_server_ts: 1_000,
+            type: "m.room.message"
+        });
+        const p = env.ircMock._whenClient(roomMapping.server, testUser.nick, "say",
+            (client, channel, text) => {
+                expect(client.nick).toEqual(testUser.nick);
+                expect(client.addr).toEqual(roomMapping.server);
+                expect(channel).toEqual(roomMapping.channel);
+                expect(text).toContain('Line one');
+                expect(text).toContain('Line one');
```
Comment (Half-Shot) [inline spec/integ/matrix-to-irc.spec.js:556]: Oh, except this.

Comment (tadzik) [reply spec/integ/matrix-to-irc.spec.js]: Ugh. Another example of why the first step of TDD is what it is :P

Thankfully it was working anyway. Amended and forcepushed, should be good now.

**Comment by Half-Shot** [comment]
Tick tick

**Pull request merged**